### PR TITLE
Editorial mdlint and vale updates to Howto/Backup directory

### PR DESCRIPTION
**Pull request by grantnoble**
Line length <= 100
EOL spaces removed
Sentence caps for headings
Active voice
Misc rewording and punctuation

### CODE REVIEW

File: docs/HowTo/Backup/Backup.md [hunk 1]
```diff
@@ -1,57 +1,53 @@
...
+description: Backing up and restoring Besu
 <!--- END of page meta data -->
 
-# Backups 
+# Backups
 
-In a decentralized blockchain, data is replicated on other nodes so it cannot be lost. However, backing up 
-configuration and data ensures a smoother recovery.  
+In a decentralized blockchain, data replicates between nodes so it is not lost, but backing up
+configuration and data ensures a smoother recovery from corrupted data or other failures.
```
Comment (MadelineMurray) [inline docs/HowTo/Backup/Backup.md]: This is a pretty long sentence now - 29 words.  I'd probably break it into two but will leave it to you to decide.

File: docs/HowTo/Backup/Backup.md [hunk 2]
```diff
@@ -1,57 +1,53 @@
...
+We recommend mounting a
+[separate volume to store data](../Get-Started/Run-Docker-Image.md#starting-besu). Use the
+[`--data-path`](../../Reference/CLI/CLI-Syntax.md#data-path) command line option to pass the path
+to Besu.
 
-The default data location is the directory in which Besu is installed
-or `/opt/besu/database` if using the [Besu Docker image](../../HowTo/Get-Started/Run-Docker-Image.md).
+The default data location is the Besu installation directory, or `/opt/besu/database` if using the
+[Besu Docker image](../../HowTo/Get-Started/Run-Docker-Image.md).
+Having some data reduces the time to synchronise a new node. You can perform periodic backups of
```
Comment (MadelineMurray) [inline docs/HowTo/Backup/Backup.md]: Should there be a line between these two paragraphs?

File: docs/HowTo/Backup/Backup.md (hunk 1 above)
Comment (grantnoble) [reply docs/HowTo/Backup/Backup.md]: Yes. Updated.

File: docs/HowTo/Backup/Backup.md (hunk 2 above)
Comment (grantnoble) [reply docs/HowTo/Backup/Backup.md]: Good pick up.

**Pull request merged**